snapshot-labs/snapshot
# feat: use stamp to resolve address to handle

### Summary

From https://github.com/snapshot-labs/pitches/issues/58

Fix #4334 
Fix #4055 

### How to test

1. ~~Wait for https://github.com/snapshot-labs/stamp/pull/113 to be merged~~
2. Go to https://snapshot-git-resolve-address-domains-using-stamp-snapshot.vercel.app/#/polls.lenster.xyz/proposal/0xda179d73d85797602a8f788eb8a326cda541a9a23d6b6b6e785682f7a98ec93e
3. It should resolves ENS and LENS addresses, with a single call to stamp API

### To-Do
